kube/Cargo.toml

```diff
@@ -33,16 +33,16 @@ features = ["derive", "client", "ws", "oauth", "jsonpatch", "admission"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-base64 = "0.13.0"
+base64 = { version = "0.13.0", optional = true }
 chrono = { version = "0.4.19", optional = true }
 dirs = { package = "dirs-next", optional = true, version = "2.0.0" }
 serde = { version = "1.0.118", features = ["derive"] }
 serde_json = "1.0.61"
-serde_yaml = "0.8.17"
+serde_yaml = { version = "0.8.17", optional = true }
 http = "0.2.2"
 url = "2.2.0"
 log = "0.4.11"
```

Comment:
I think `log` is no longer used directly (`kube` and `kube-core`). Replaced by `tracing`.

Reply:
There's a naked `warn!` in `params.rs` and some naked `debug!` and `trace!` calls in `config`, but can fix this.

Reply:
have removed it from kube-core at least

Reply:
Ah, I missed those. `tracing` acts like `log` without subscribers, so replacing with `tracing` equivalent shouldn't affect users.

Reply:
swapped over to tracing in `kube`